closure/repositories.bzl

```diff
@@ -56,15 +56,17 @@ def closure_repositories():
 
   native.maven_jar(
       name = "closure_compiler",
-      artifact = "com.google.javascript:closure-compiler:v20160208",
-      sha1 = "5a2f4be6cf41e27ed7119d26cb8f106300d87d91",
+      artifact = "com.google.javascript:closure-compiler:v20160315",
+      sha1 = "f5b1a03f83a014e545db60a795fcf94db14a5ba2",
   )
 
+  # To update Closure Library, one needs to uncomment and run the js_files_maker
+  # and js_testing_files_maker genrules in closure_library.BUILD.
   native.new_http_archive(
       name = "closure_library",
-      url = "https://bazel-mirror.storage.googleapis.com/github.com/google/closure-library/archive/20160208.zip",
-      sha256 = "8f610300e4930190137505a574a54d12346426f2a7b4f179026e41674e452a86",
-      strip_prefix = "closure-library-20160208",
+      url = "https://github.com/google/closure-library/archive/v20160315.zip",
```

Comment:
As an aside, why do the names of these rules use underscores like `closure_library`, instead of hyphens like `closure-library`?